README.md

```diff
@@ -1,23 +1,75 @@
-Python courses for the scientific researcher
-============================================
+# Python courses for the scientific researcher
 
 This repository contains several courses for the benefit of scientific researchers,
-particularly in the fields of oceanography and meteorology.
+particularly in the fields of oceanography and meteorology.  
+All the content is fully open-source and freely downloadable.
 
-There are currently four courses:
+These courses are presented as interactive self-learning tutorials,
+contained in ["Jupyter notebooks"](https://jupyter.org/index.html).
+The user will run the notebook interactively, and will also edit it and run
+their own code, particularly to complete the set learning exercises.
 
-### An introduction to numpy
-3.5 hours &mdash; depends on a basic Python background\
-https://mybinder.org/v2/gh/SciTools/courses/master?filepath=course_content%2Fextra_courses%2Fnumpy_intro.ipynb
+The most flexible learning method is to download this repository and run it in a
+Python environment with the correct dependencies.  
+A suitable environment can be produced with
+[conda](https://conda.io/projects/conda/en/latest/user-guide/getting-started.html)
+(only 'miniconda' is needed).  
+Use a command such as
```

Comment:
I also think this needs to be more definite.  I'd remove 'such as' because it introduces vagueness.